commands.py

```diff
@@ -0,0 +1,57 @@
+import config
+import discord
+from discord.ext import commands
+import sqlite3
+
+def add_username(irl, game, username):
+    try:
+        with conn:
+            c.execute("""INSERT INTO users (irl, game, username)
+                      VALUES (?, ?, ?)
+                      """, (irl.lower(), game.lower(), username.lower()))
+        await ctx.send('Added {} to {} for user {}'.format(args[2], args[1], args[0]))
+    except:
+        await ctx.send('Invalid arguments! Try checking to make sure it\'s $dnsbot {user} {game} {username}')
+
+def grab_username(irl, game):
+    try:
+        with conn:
+            c.execute("""SELECT username FROM users
+                      WHERE irl=? AND game=? 
+                      """, (irl.lower(), game.lower()))
+        response = c.fetchone()[0]

```

Comment:
If there are no matches I would assume the returned array is empty so doing `[0]` would throw an `IndexError`